bzt/modules/gatling.py

```diff
@@ -230,7 +229,7 @@ def __get_res_files_from_script(script_contents):
                 param_list = found_sample.split(",")
                 param_index = 0 if "separatedValues" in search_pattern.pattern else -1  # first or last param
                 file_path = re.compile(r'\".*?\"').findall(param_list[param_index])[0].strip('"')
-                resource_files.append(file_path)
+                resource_files.append(self.engine.find_file(file_path))
```

Comment:
find_file should be applied only to file paths passed from config. Inner files parsed out from other files should not be affected.